---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}


# My CV

## View or Download My CV
You can view my CV below or [download it here](resume.pdf).

<iframe src="files/CV_Vinay_Rajput_ 2023_v2.pdf" width="100%" height="600px" style="border: none;"></iframe>


# Vinay Rajput

**Location**: Pune, India  
**Mobile No**: +91-8447432515  
**Email**: [srm.vinay0005@gmail.com](mailto:srm.vinay0005@gmail.com), [v.rajput@ncl.res.in](mailto:v.rajput@ncl.res.in)  
**Google Scholar**: [https://scholar.google.com/citations?user=YnIqsiAAAAAJ&hl=en](https://scholar.google.com/citations?user=YnIqsiAAAAAJ&hl=en)  
**ResearchGate**: [https://www.researchgate.net/profile/Vinay-Rajput-2](https://www.researchgate.net/profile/Vinay-Rajput-2)  
**GitHub**: [https://github.com/vinayrajput0005](https://github.com/vinayrajput0005)  

---

## Summary

Experienced bioinformatics Analyst specialized in NGS data analysis, proficient in Python and R, with a strong track record of delivering impactful insights from complex genomic datasets. Skilled in designing bioinformatics pipelines, applying statistical models, and collaborating with interdisciplinary teams to drive successful outcomes. Passionate about leveraging cutting-edge technologies to contribute to personalized medicine and precision healthcare.

---

## Research Experience

**Senior Project Associate**  
*National Collection of Industrial Microorganisms, CSIR-National Chemical Laboratory*  
_February 2023 – Present_  
**Supervisor**: Dr. Mahesh Dharne  
**Roles & Responsibilities**:
- Carried out Data Analysis on multiple NGS sequences like 16s/18s/ITR amplicons, shotgun metagenomes, and whole genomes (bacteria/viruses).
- Sampling, DNA extraction, library preparation, and sequencing using Illumina (MiSeq/HiSeq) and Oxford Nanopore (MINion).
- Set up and managed Linux-based high-performance computing infrastructure, created bioinformatics pipelines, and provided guidance to PhD students.

**Project Assistant-III**  
*National Collection of Industrial Microorganisms, CSIR-National Chemical Laboratory*  
_October 2020 – February 2023_  
**Supervisor**: Dr. Mahesh Dharne  

**Project Assistant-II**  
*National Collection of Industrial Microorganisms, CSIR-National Chemical Laboratory*  
_October 2018 – October 2020_  
**Supervisor**: Dr. Mahesh Dharne  

**Project Trainee**  
*CSIR-Center for Cellular and Molecular Biology, Hyderabad*  
_September 2018 – October 2018_  
**Supervisor**: Dr. Shrish Tiwari  

---

## Education

- **Full Stack Data Science Program**, iNeuron-Bengaluru, Nov 2021 – November 2022  
- **Python Online Certification Training**, April 2020  
- **B.Tech in Bioinformatics**, SRM University, Delhi-NCR, 2014 – 2018 (CGPA: 7.85/10, 78.5%)  
- **Higher Secondary Schooling (Science)**, Government Boys Senior Secondary School, Alipur Delhi (73.40%), 2014  
- **Secondary Schooling**, Bhagatji Memorial Model Secondary School, Delhi (CGPA: 7.2), 2012  

---

## Research Projects

- **Wastewater Surveillance of Covid-19 & Other Bacterial and Viral Pathogens** from Pune city under the One Health Approach (Funding: Rockefeller Foundation, USA & CSIR).
- **River Microbiome**: Analyzing microbial ecology of polluted urban rivers and the impact of mass gathering events on India’s rivers.
- **National Mission for Clean Ganga (NMCG)**: Studying non-putrefying properties of the Ganges river using Nanopore Sequencing.
- **Lonar Microbial Ecology**: Understanding microbial flora in the unique soda lake in Maharashtra, India.
- **Whole Genome Sequencing of Ochrobactrum species** using Oxford Nanopore technology for genome assembly, gene prediction, and phylogenetic analysis.

---

## Professional Skills

**Dry Lab**:
- Sequence Analysis: BLAST, clustalW, Muscle, FASTA, Diamond.
- Metagenomics Tools: QIIME, MOTHUR, MG-RAST, MEGAN, DeepARG, Ressfinder, Kaiju.
- Pathway Databases: KEGG, MINT, BIND.
- Genomic Annotation: PATRIC, Prokka, RAST.
- Assembly Tools: SPADES, Flye, Canu, Unicycler, wdbg2.
- Proficient in scripting and analysis pipeline development (Snakemake, Nextflow).

**Computer Skills**:
- **Operating Systems**: Linux, Windows.
- **Programming**: Python (NumPy, SciPy, Pandas), R, Shell Scripting.
- **Visualization**: Matplotlib, Seaborn.
- **Miscellaneous**: Advanced awk, sed, grep; proficient in Microsoft Office.

**Wet Lab**:
- Microbiological Techniques: Plating, Streaking, Staining, Culture requirements.
- DNA Sequencing: Oxford Nanopore.
- Molecular Biology: DNA extraction, Amplification (PCR), Library Preparation.

---

## Area of Interest

- Programming, Metagenomics, Single-Cell Genomics, Epigenetics, Cancer Genomics, Microbial Genomics, Bioinformatics Pipeline Development, Oxford Nanopore Sequencing.

---

## Publications

1. **Samson, R., Rajput, V., Yadav, R., Shah, M., Dastager, S., Khairnar, K., Dharne, M.** (2023). Spatio-temporal variation of the microbiome and resistome repertoire along an anthropogenically dynamic segment of the Ganges River, India. *Science of The Total Environment*, 872, 162125. [DOI](https://doi.org/10.1016/j.scitotenv.2023.162125)

2. **Malik, V., Rajput, V., Pramanik, R., Samson, R., Yadav, R. Kumar, Kadam, P., Shah, N., Sawant, R., Bhalerao, U., Tupekar, M., Khan, S., Shah, P., Shashidhara, L., Kamble, S., Dastager, S., Karmodiya, K., Dharne, M.** (2023). Campus Sewage Water Surveillance based dynamics and infection trends of SARS-CoV-2 variants during third wave of COVID-19 in Pune, India. Cold Spring Harbor Laboratory.

3. **Yadav, R., Rajput, V., Dharne, M.** (2023). Water Hyacinth microbiome: metagenomic cues from environment and functionality in urban aquatic bodies. Cold Spring Harbor Laboratory.

4. **Rajput, V., Pramanik, R., Malik, V., Yadav, R., Kadam, P., Bhalerao, U., Tupekar, M., Deshpande, D., Shah, P., Shashidhara, L., Boargaonkar, R., Patil, D., Kale, S., Bhalerao, A., Jain, N., Kamble, S., Dastager, S., Karmodiya, K., Dharne, M.** (2023). Genomic surveillance reveals early detection and transition of Delta to Omicron Lineages of SARS-CoV-2 Variants in wastewater treatment plants of Pune, India. Cold Spring Harbor Laboratory.

5. **Prabhu, V.A., Rajput, V.** (n.d.). Gut Microbiota Dysbiosis in Patients with Intracranial... *Annals of Indian Academy of Neurology*. [Link](https://journals.lww.com/annalsofian/Fulltext/2022/25050/Gut_Microbiota_Dysbiosis_in_Patients_wit.56.aspx)

6. **Rajput, V., Samson, R., Yadav, R., Dastager, S., Khairnar, K., Dharne, M.** (2022). Metagenomic mining of Indian river confluence reveal functional microbial community with lignocelluloytic potential. *3 Biotech*, 12, 1–10. [DOI](https://doi.org/10.1007/s13205-022-03190-7)

7. **Dharmadhikari, T., Rajput, V., Yadav, R., Boargaonkar, R., Patil, D., Kale, S., Kamble, S.P., Dastager, S.G., Dharne, M.S.** (2022). High throughput sequencing based direct detection of SARS-CoV-2 fragments in wastewater of Pune, West India. *Science of The Total Environment*, 807, 151038. [DOI](https://doi.org/10.1016/j.scitotenv.2021.151038)

8. **Dharmadhikari, T., Rajput, V., Yadav, R., Boargaonkar, R., Patil, D., Kale, S., Kamble, S.P., Dastager, S.G., Dharne, M.S.** (2022). High throughput sequencing based direct detection of SARS-CoV-2 fragments in wastewater of Pune, West India. *Science of The Total Environment*, 807, 151038.

9. **Rajput, V., Yadav, R., Dharne, M.** (2022). Metagenomic exploration reveals a differential patterning of antibiotic resistance genes in urban and peri-urban stretches of a riverine system. *Environmental Science and Pollution Research*, 28(46), 66477-66484.

10. **Yadav, R., Rajput, V., Dharne, M.** (2022). Metagenomic analysis of a mega-city river network reveals microbial compositional heterogeneity among urban and peri-urban river stretch. *Science of The Total Environment*, 783, 146960.

11. **Mohapatra, M., Yadav, R., Rajput, V., Dharne, M., Rastogi, G.** (2022). Metagenomic analysis reveals genetic insights on biogeochemical cycling, xenobiotic degradation, and stress resistance in mudflat microbiome. *Journal of Environmental Management*, 292, 112738.

12. **Chakraborty, J., Rajput, V., Sapkale, V., Kamble, S., Dharne, M.** (2022). Spatio-temporal resolution of taxonomic and functional microbiome of Lonar soda lake of India reveals metabolic potential for bioremediation. *Chemosphere*, 264, 128574.

13. **Yadav, R., Rajput, V., Dharne, M.** (2022). Functional metagenomic landscape of polluted river reveals potential genes involved in degradation of xenobiotic pollutants. *Environmental Research*, 192, 110332.

14. **Yadav, R., Rajput, V., Gohil, K., Khairnar, K., Dharne, M.** (2022). Comprehensive metagenomic insights into a unique mass gathering and bathing event reveals transient influence on a riverine ecosystem. *Ecotoxicology and Environmental Safety*, 202, 110938.

15. **Gohil, K., Rajput, V., Dharne, M.** (2022). Pan-genomics of Ochrobactrum species from clinical and environmental origins reveals distinct populations and possible links. *Genomics*, 112(5), 3003-3012.

16. **Samson, R., Rajput, V., Shah, M., Yadav, R., Sarode, P., Dastager, S.** (2022). Deciphering taxonomic and functional diversity of fungi as potential bioindicators within confluence stretch of Ganges and Yamuna Rivers, impacted by anthropogenic activities. *Chemosphere*, 252, 126507.

17. **Chakraborty, J., Sapkale, V., Rajput, V., Shah, M., Kamble, S., Dharne, M.** (2022). Shotgun metagenome guided exploration of anthropogenically driven resistomic hotspots within Lonar soda lake of India. *Ecotoxicology and Environmental Safety*, 194, 110443.

18. **Chakraborty, J., Sapkale, V., Shah, M., Rajput, V., Mehetre, G., Agawane, S.** (2022). Metagenome sequencing to unveil microbial community composition and prevalence of antibiotic and metal resistance genes in hypersaline and hyperalkaline Lonar Lake, India. *Ecological Indicators*, 110, 105827.

19. **Zothanpuia, A.K., Passari, P., Deka, P., Rajput, V., Priya, L.P.M., Dharne, M.** (2020). Draft Genome Sequence of Freshwater-Derived *Streptomyces* sp. Strain BPSDS2, Isolated from Damte Stream, Northeast India. *Microbiology Resource Announcements*, 8(43), e00874-19.

20. **Samson, R., Shah, M., Yadav, R., Sarode, P., Rajput, V., Dastager, S.** (2021). Metagenomic insights to understand transient influence of Yamuna River on taxonomic and functional aspects of bacterial and archaeal communities of River Ganges. *Science of the Total Environment*, 674, 288-299.

21. **Passari, A.K., Rajput, V., Zothanpuia, L.P.M., Priya, S. Dastager, M.** (2020). Draft Genome Sequence of *Streptomyces thermocarboxydus* BPSAC147, a Potentially Plant Growth-Promoting Endophytic Bacterium. *Microbiology Resource Announcements*, 8(23), e00363-19.

22. **Passari, A.K., Rajput, V., Zothanpuia, L.P.M.** (2020). Draft Genome Sequence of Plant Growth-Promoting Endophytic *Microbacterium hydrothermale* BPSAC84, Isolated from the Medicinal Plant *Mirabilis jalapa*. *Microbiology Resource Announcements*, 8(23), e00363-19.

---

## TRAININGS & CONFERENCES

- 5-Days Industrial training at International Center for Genetic Engineering and Biotechnology (ICGEB), New-Delhi, India.
- Completed 2-Months Summer training at CSIR-Center for Cellular and Molecular Biology, Hyderabad under the guidance of Dr. Shrish Tiwari. From 25th May 2017 to 25th July 2017.
- Dissertation project training for 6-Months at CSIR-Center for Cellular and Molecular Biology, under the guidance of Dr. Shrish Tiwari. From 4th Jan 2018 to 4th July 2018.
- Participated in International conference on Antimicrobial Resistance, Nobel Drug Discovery and Development: Challenges and Opportunities, organized by SRM University, Delhi- NCR. Year: 2015.
- Participated in International conference on “Integrative Biology & Applied Genetics”- ICIBAG: 2018, organized by Osmania University, Hyderabad. Year: March-2018.
- Participated in 9th International Conference on “NextGen Genomics, Biology, Bioinformatics and Technologies (2019-NGBT)”. Date: 30th Sep – 2nd Oct 2019, Mumbai, India.
- Participated in organizing Skill Development Program conducted at CSIR-National Chemical Laboratory, Pune. Title: Microbial Identification by Biochemical, Genetics and Genomic Techniques. From: 18th June 2018 to 28th June 2018.


